### ISSUE TITLE
Config.unobserve is deprecated.

### ISSUE THREAD

**Issue by k2b6s9j** [gepoch/linter-write-good#5]
Config::unobserve no longer does anything. Call `.dispose()` on the object returned by Config::observe instead.
```
Config.unobserve (/Applications/Atom.app/Contents/Resources/app/src/config.js:374:19)
LinterWriteGood.destroy (/Users/kepler/.atom/packages/linter-write-good/lib/linter-write-good.coffee:52:16)
```

Sorry about the cookie-cutter issue, just updated the package to 0.3.0 and this popped up at startup.

**Comment by gepoch** [comment]
Thanks for the ticket!

**Comment by gepoch** [comment]
Fixed in 0.3.1
